peps/pep-9999.rst

```diff
@@ -267,6 +637,8 @@ If the user wants to use Conda packages and the ``mamba`` package manager to
 install external dependencies, they may specify that in a
 ``~/.pypi-name-mappings`` file:
 
+.. JRG: Should this file use a XDG-standardized location?
```

Comment:
Connected to the comment higher up about sharing per ecosystem

Reply:
Not sure I'm following this bit 😬 Can you elaborate?

Reply:
Connected to:

What I'm saying is that your assumption here may not be correct. E.g, pixi has its own mapping of PyPI to conda-forge, not shared with other tools, I believe. Similarly, I have no idea whether yum/dnf are sharing a mapping (or would be willing to). When there are multiple package managers, they tend to compete rather than collaborate closely.